### ISSUE TITLE
Prepare tools for the collaborative phase

### ISSUE THREAD

**Issue by tschaffter** [data2health/DREAM-Challenge#70]
- [ ] Create Slack workspace / channel to interact with participants
- [x] Create Zoom calendar invite
- [x] Create shared Google folder
- [x] Create Google doc for the agenda
- [x] Create Google sheet for list of members and contact info

**Comment by mellybelly** [comment]
@tschaffter consider coordinating with @andrewjneumann to coalesce things/double file for easy navigation in N3C space. we want to encourage participation!

**Comment by tschaffter** [comment]
## Update

- Zoom meeting: waiting on reply from Xengie regarding use of Zoom componc account
- Will ask participants for the email addresses that we should use to invite them to the Google folder. Tim will have to set Sharing permission as I don't have them (UW G Drive).

**Comment by tschaffter** [comment]
Hi @andrewjneumann, see Melissa's message above. Last week, we have officially started the collaborative phase of the [EHR DREAM Challlenge: Patient Mortality Prediction](https://www.synapse.org/ehr_dream_challenge_mortality). In this phase, 14 developers of EHR models from the top five best-performing teams in the competitive phase and the organizers of the challenge will collaborate on analyzing the results, performing additional experiments and writing the challenge manuscript. Please let me know if you would like to invite the participants to the N3C space.

**Comment by andrewjneumann** [comment]
@tschaffter Yes! Thank you, I think it'd be great to include a results discussion. Happy to share N3C info: https://covid.cd2h.org/N3C_analytics (Clinical Scenarios & Data Analytics subgroup & https://covid.cd2h.org/N3C_synthetic_data are of probably the most interest for folks)... if there's a mailing list or a standing meeting of folks happy to chat and answer questions.
